lexicons/com/atproto/admin/defs.json

```diff
@@ -587,6 +587,23 @@
         }
       }
     },
+    "modEventFlag": {
```

Comment:
would definitely add optional `comment` field; I think every event type should have `comment`